HomeWork3/Library.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace HomeWork3
+{
+    class Library
+    {
+        private List<Book> _books;
+        public Library()
+        {
+            _books = new List<Book>
+            {
+                new Book("William", "Shakespeare", "Hamlet", "some text"),
```

Comment:
please, try to put 2 variants autor as objects, and 2 more as: FirstName + 1 LastName